frontend/src/utils/helpers/moduleUtils.js

```diff
@@ -23,6 +23,12 @@ function isModuleEnabled(module) {
   if (module === ModuleNames.SHARES) {
     return getModuleActiveStatus(ModuleNames.DATASETS);
   }
+  if (module === ModuleNames.WORKSHEETS) {
+    return (
+      getModuleActiveStatus(ModuleNames.DATASETS) &&
+      getModuleActiveStatus(ModuleNames.WORKSHEETS)
+    );
+  }
```

Comment:
I think is is out of scope for V2.0.0 release, but we need a better way of managing dependencies in the frontend. Wdyt?

Reply:
Yes, I believe we discussed this earlier in the module enablement PR.